packages/flutter/lib/src/material/chip.dart

```diff
@@ -74,6 +78,18 @@ class Chip extends StatelessWidget {
   /// The delete button is included in the chip only if this callback is non-null.
   final VoidCallback onDeleted;
 
+  /// The style to be used for the label text.
+  final TextStyle labelStyle;
+
+  /// Message for the chip delete button's tooltip.
+  final String deleteButtonTooltipMessage;
+
+  /// Background color for the widget.
```

Comment:
These docs aren't detailed enough. We can guess that a property called "backgroundColor" on a widget gives the background color of the widget. :-)

What it should say is where the color appears, what the default is, etc.